lib/utils/setupDB.js

```diff
@@ -0,0 +1,26 @@
+const { reviewer, studio } = require('./makeFakeData.js');
+const pool = require('./pool.js');
+
+module.exports = async () => {
+  for (let i = 0; i < 5; i++) {
+    const fakeReviewer = reviewer();
+    await pool.query('INSERT INTO reviewers (name, company) VALUES ($1, $2);', [
+      fakeReviewer.name,
+      fakeReviewer.company,
+    ]);
+  }
+
+  for (let i = 0; i < 3; i++) {
+    const fakeStudio = studio();
+    await pool.query(
+      `
+      INSERT INTO studios (
+        name,
+        city,
+        state,
+        country
+      ) VALUES ($1, $2, $3, $4);`,
+      [fakeStudio.name, fakeStudio.city, fakeStudio.state, fakeStudio.country]
```

Comment:
Totally just a nitpick here because you all CRUSHED this, but when you are pulling multiple parameters from the same object, you can simply destructure it in your const declaration. 

For example: `const {name, city, state, country} = studio()` 

Then you don't have to repeat `fakeStudio.___` multiple times in your insert.